cpu/cpu_linux.go

```diff
@@ -152,9 +152,9 @@ func Info() ([]InfoStat, error) {
 				return ret, err
 			}
 			c.Stepping = int32(t)
-		case "cpu MHz":
+		case "cpu MHz", "clock":
 			// treat this as the fallback value, thus we ignore error
-			if t, err := strconv.ParseFloat(value, 64); err == nil {
+			if t, err := strconv.ParseFloat(strings.Replace(value, "MHz", "", 1), 64); err == nil {
```

Comment:
What happens if the unit is not "MHz", but, say, "kHz" or "GHz"? Since I don't have any experience with PPC/Power architecture, I don't know. Of course, "cpu kHz" can be a problem too, but I have not seen that.